one-ui/src/main/java/com/marklogic/hub/oneui/Application.java

```diff
@@ -29,9 +30,9 @@
 import java.util.Set;
 import java.util.stream.Stream;
 
-@ComponentScan(basePackages = "com.marklogic.hub", excludeFilters = {@ComponentScan.Filter(
-		type = FilterType.REGEX,
-		pattern = "com\\.marklogic\\.hub\\.impl\\.HubConfigImpl")
+@ComponentScan(basePackages = "com.marklogic.hub",
+    excludeFilters = {@ComponentScan.Filter(type = FilterType.REGEX, pattern = "com\\.marklogic\\.hub\\.impl\\.HubConfigImpl"),
+    @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = HubTestBase.class)
```

Comment:
@rjrudin ,
The only way I can prevent spring container from instantiating HubTestBase and it's derived classes is to exclude it here. I don't feel right as one-ui code has dependency on test code in marklogic-data-hub. Is that ok or is there a better way to prevent its instantiation